docs/fedora-releases-guide/releasing-multiple-packages.md

```diff
@@ -0,0 +1,198 @@
+
+## Simplest scenario
+
+Let's have two packages, `foo` and `bar`, that we want to release together in one update.
+The packages build independently of each other.
+
+Here are the relevant parts from Packit config of `foo`:
+
+```yaml
+downstream_package_name: foo
+
+jobs:
+  - job: koji_build
+    trigger: commit
+    sidetag_group: foo-bar-releases
+    dependents:
+      - bar
+    dist_git_branches:
+      - fedora-all
+```
+
+Here are the relevant parts from Packit config of `bar`:
+
+```yaml
+downstream_package_name: bar
+
+jobs:
+  - job: koji_build
+    trigger: commit
+    sidetag_group: foo-bar-releases
+    dist_git_branches:
+      - fedora-all
+
+  - job: bodhi_update
+    trigger: koji_build
+    sidetag_group: foo-bar-releases
+    dependencies:
+      - foo
+    dist_git_branches:
+      - fedora-all
+```
+
+Note there is only one `bodhi_update` job for both packages, we chose to configure it in `bar`.
+For that reason we have to reference `bar` from `dependents` of the `koji_build` job in `foo`,
+and, vice versa, reference `foo` from `dependencies` of the `bodhi_update` job in `bar`.
+The `sidetag_group` option, having the same value in all jobs, defines the sidetags Koji builds
+will be tagged into and Bodhi update will be created from. It doesn't matter which package
+is built first, only after builds of both of them are tagged into the sidetag a Bodhi update
+is created and the sidetag removed.
+
+## Ensuring build order
+
+Let's have the same two packages as before, but this time `bar` depends on `foo`, meaning that
+a build of `bar` can fail if it is build before `foo`. So we have to make sure that `foo` is always
+built first.
+
+Here are the relevant parts from Packit config of `foo`:
+
+```yaml
+downstream_package_name: foo
+
+jobs:
+  - job: koji_build
+    trigger: commit
+    sidetag_group: foo-bar-releases
+    dependents:
+      - bar
+    dist_git_branches:
+      - fedora-all
+```
+
+Here are the relevant parts from Packit config of `bar`:
+
+```yaml
+downstream_package_name: bar
+
+jobs:
+  - job: koji_build
+    trigger: commit | koji_build
+    sidetag_group: foo-bar-releases
+    dependencies:
+      - foo
+    dist_git_branches:
+      - fedora-all
+
+  - job: bodhi_update
+    trigger: koji_build
+    sidetag_group: foo-bar-releases
+    dependencies:
+      - foo
+    dist_git_branches:
+      - fedora-all
+```
+
+Note that config of `foo` hasn't changed at all. The `koji_build` job in `bar` however now has
+two triggers - it will be triggered also when a build of `foo` lands in the common sidetag.
+It also has `foo` in `dependencies`, so if it is triggred via commit while `foo` is not yet
+in the sidetag, it will be skipped. As before, once builds of both packages are in the sidetag,
+a Bodhi update is created and the sidetag removed.
+
+## Skipping release of some packages
+
+But what if we only want to release `bar` and not `foo`? We can tag the latest stable build of `foo`
```

Comment:
```suggestion
What if we only want to release `bar` and not `foo`? We can tag the latest stable build of `foo`
```

I would not start a new paragraph with *but*.

Reply:
I was trying to avoid something like `That's all cool and dandy, but what if ...` :sweat_smile: